### ISSUE TITLE
Cannot read properties of null (reading 'listen') for vitest & storybook

### ISSUE THREAD

**Issue by ck-coatue** [sumy7/vite-plugin-generate-file#3]
I've noticed that if vite-plugin-generate-file is used in a vite project and that vite project also has vitest and/or storybook installed, the below error appears when running vitest or storybook (the one below is from storybook but it's essentially the same error in vitest).

```
TypeError: Cannot read properties of null (reading 'listen')
    at b (./node_modules/vite-plugin-generate-file/dist/index.cjs.js:1:2145)
    at _createServer (file://./node_modules/vite/dist/node/chunks/dep-2b82a1ce.js:65227:30)
    at async Module.start (./node_modules/@storybook/builder-vite/dist/index.js:159:12520)
    at async storybookDevServer (./node_modules/@storybook/core-server/dist/index.js:104:2460)
    at async buildDevStandalone (./node_modules/@storybook/core-server/dist/index.js:119:3007)
    at async withTelemetry (./node_modules/@storybook/core-server/dist/index.js:103:3903)
    at async dev (./node_modules/@storybook/cli/dist/generate.js:473:401)
    at async Command.<anonymous> (./node_modules/@storybook/cli/dist/generate.js:475:225)
```

If I remove generateFile from vite.config.ts, everything works.

I guess it's something to do with the devServer not being started when vitest or storybook is started.

**Comment by sumy7** [comment]
Hi @ck-coatue. Thank you for your feedback. I will investigate the cause of this issue.
Could you please provide the versions of Vite, Vitest, and Storybook you are using? 
I suspect that the version of the plugin dependency might be outdated.

**Comment by sumy7** [comment]
I have released vite-plugin-generate-file@0.1.1, in which upgraded the Vite dependency to version 4.5.0.
Could you please try this new version and let me know if you are still experiencing the issue?

**Comment by ck-coatue** [comment]
nice, that worked 😄 thank you!
